src/receive_imf/tests.rs

```diff
@@ -5070,32 +5042,6 @@ async fn test_unarchive_on_member_removal() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-async fn test_no_op_member_added_is_trash() -> Result<()> {
```

Comment:
Btw, maybe such no-op message should be just hidden? If they are visible, the user could think that the member indeed wasn't in the group. Hidden messages don't break assigning subsequent messages to the chat by References